tests/main/interfaces-alsa/task.yaml

```diff
@@ -0,0 +1,94 @@
+summary: Ensure that the alsa interface works.
+
+systems: [-debian-*]
```

Comment:
Yes, there is no confinement on debian. That is what I introduced the `snap debug confinement = "strict"` check to all other interface tests to split them into two portions, one for the allowed part (as there is still seccomp in place and a few other things worth to check) and one for the disallowed path. The disallowed part is always skipped when `snap debug confinement` returns `none`. See https://github.com/snapcore/snapd/blob/master/tests/main/interfaces-firewall-control/task.yaml#L72 for an example.